hy/completer.py

```diff
@@ -28,8 +28,13 @@
 import os
 import re
 import sys
+import hy.macros
+import hy.compiler
+
 from contextlib import contextmanager
```

Comment:
Could you change this to `import contextlib` and update its usage to `@contextlib.contextmanager` in line 131?